packages/ui/src/components/navbar/navbar-user/index.tsx

```diff
@@ -107,15 +113,41 @@ export const NavbarUser = ({
           </DropdownMenuSubTrigger>
           <DropdownMenuPortal>
             <DropdownMenuSubContent>
+              <DropdownMenuItem disabled>Color Scheme</DropdownMenuItem>
+              <DropdownMenuSeparator />
               <DropdownMenuRadioGroup
-                value={theme}
+                value={mode}
                 onValueChange={value => {
-                  setTheme(value as FullTheme)
+                  setTheme(`${value}-${color}-${contrast}` as FullTheme)
```

Comment:
Looks like this is repeated a few times, might be good to extract a small utility function

```typescript
function formatFullTheme(theme: string, color: string, contrast: string): FullTheme {
  return `${theme}-${color}-${contrast}` as FullTheme
}
```

Reply:
the function param is different in all three places, so we'll end up with the same duplication again 🤔